app/views/about/index.html.erb

```diff
@@ -126,14 +126,14 @@
       <ul>
         <li><%= link_to _('API Resources'), apipie_apipie_path, :title => _('Automate Foreman via a simple and powerful API') %></li>
         <li><%= link_to _('Manual'), documentation_url, :rel => "external" %></li>
-        <li><%= link_to _('Wiki'), "http://projects.theforeman.org", :rel => "external" %></li>
+        <li><%= link_to _('Wiki'), external_link_path(type: 'wiki'), :rel => "external" %></li>
       </ul>
       <h6><%= _("IRC") %></h6>
-      <p><%= (_("You can find The Foreman on the %{freenode} (irc.freenode.net) network.  For general support, please visit #theforeman and for development specific related chat, please visit #theforeman-dev.") % {:freenode => link_to("freenode", "http://www.freenode.net", :rel => "external")}).html_safe  %></p>
+      <p><%= (_("You can find The Foreman on the %{freenode} (irc.freenode.net) network.  For general support, please visit #theforeman and for development specific related chat, please visit #theforeman-dev.") % {:freenode => link_to("freenode", external_link_path(type: 'chat'), :rel => "external")}).html_safe  %></p>
```

Comment:
I'd name this `freenode` since the hostname (`irc.freenode.net`) is hardcoded in the translation string as well. Feels to name it chat given the header is also `IRC`. Sure, it can be changed via translations but then you have yet another thing to translate very illogically.

```suggestion
      <p><%= (_("You can find The Foreman on the %{freenode} (irc.freenode.net) network.  For general support, please visit #theforeman and for development specific related chat, please visit #theforeman-dev.") % {:freenode => link_to("freenode", external_link_path(type: 'freenode'), :rel => "external")}).html_safe  %></p>
```

Reply:
I think the link should remain general. As for the text - there is a way to override the translated text, so this way the text and the link will be redirected to a different location.